app/views/tags/show.html.erb

```diff
@@ -21,8 +21,13 @@
   <!--/descriptions-->
 
   <!--main content-->
-  <p><%= ts("This tag belongs to the %{tag_class} Category.", :tag_class => @tag.class::NAME) %>
+  <p>
   <% if @tag.canonical %>
+    <% if @tag.is_a?(Fandom) %>
+      This tag belongs to the Fandom Category, a list of <%= link_to ts('all the relationships is available'), fandom_path(@tag.name) %>.
```

Comment:
I'd suggest making "list of all the relationships" the link text and leaving the other words out of the link (but keep them in the sentence, of course). Also, there's a comma splice going on here -- it should be two separate sentences: "This tag belongs to the Fandom Category. A list of all the relationships is available." 

However, now I have a question! Is it a list of relationship _tags_, e.g. Kirk/Spock, or a list of tags that are related to the fandom? Maybe it should be "A list of all the Relationship tags in this fandom is available" (which I think is what's going on here) or "A list of all tags related to this Fandom is available" for clarity.